### PR TITLE
Send Windows install logs to API server

### PR DESCRIPTION
**Pull request by zanderz**
@maxtaco @patrickxb

### CODE REVIEW

**Comment by zanderz** [comment]
@maxtaco @patrickxb 
I don't get what's up with the JS tests.
Also, the vendoring for this seems overly heavy - should we do a quicker + dirtier UTF16 decode?

**Comment by maxtaco** [comment]
hi steve try a rebase, it was broken last night!

File: go/client/cmd_log_send.go [hunk 1]
```diff
@@ -75,6 +68,7 @@ func (c *CmdLogSend) Run() error {
 	}
 
 	logs := c.logFiles(status)
+	defer os.Remove(logs.Install)
```
Comment (maxtaco) [inline go/client/cmd_log_send.go]: what happens here on osx or linux?

File: go/client/cmd_log_send.go [hunk 2]
```diff
@@ -134,13 +128,15 @@ func (c *CmdLogSend) GetUsage() libkb.Usage {
 
 func (c *CmdLogSend) logFiles(status *fstatus) libkb.Logs {
 	logDir := c.G().Env.GetLogDir()
+	installLogPath, _ := GetInstallLogPath()
```
Comment (patrickxb) [inline go/client/cmd_log_send.go]: Can you check the error here?

File: go/client/cmd_log_send.go (hunk 1 above)
Comment (zanderz) [inline go/client/cmd_log_send.go]: Seems to be harmless, I tried it. Should it be changed?

Comment (maxtaco) [inline go/client/cmd_log_send.go]: How about

```go
// Some comme about how this is only going to do something on Windows etc etc 
defer func() {
   if logs.Install != "" {
    os.Remove(logs.Install)
   } 
}()

**Comment by zanderz** [comment]
How about now @maxtaco @patrickxb ?

**Comment by maxtaco** [comment]
I'm

**Comment by maxtaco** [comment]
I'm a 👍 so long as all good on linux and macOS

**Comment by patrickxb** [comment]
LGTM

**Pull request merged**